orion/sites/admin/src/announce/announce.js

```diff
@@ -13,43 +15,27 @@ export class AnnouncePage extends React.Component {
  	}
 
 	render() {
-		var numAnnouncements = 3;
-		let fakeAnnounceA = {
-			date: "02/15/2020",
-			author: "Melon King",
-			message: "Only evil people waste watermelon"
-		};
-		let fakeAnnounceB = {
-			date: "02/16/2020",
-			author: "Fruit Hater",
-			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
-		};
-		/* TODO: will use this to test text wrapping later */
-		let fakeAnnounceC = {
-			date: "02/23/2020",
-			author: "Talkative",
-			message: "This will be a longer message later"
-		};
+		const rows = this.state.list.map ((row, index) => {
+	      return <AnnounceRow key = {index} row = {row}/>
```

Comment:
nit:
just make this `key={index}` and `row={row}` (without the spaces in between). The idea is to keep the parameter groups together. So it should look like:
```
<AnnounceRow key={index} row={row}/>
```